### PR TITLE
+ Fixed DownloadIcon and guild separator to just be hidden now

### PR DESCRIPTION
**Pull request by BE-CH**
Fixed such that the Download Icon and the guild seperator are hidden now instead of removed.

### CODE REVIEW

**Comment by BE-CH** [comment]
Add support for CTRL + R such that we inject javascripts again.
Furthermore i fixed the platform-osx error.

File: index.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 const {
 	hasScreenCapturePermission,
 	openSystemPreferences,
 } = require('mac-screen-capture-permissions');
 const { autoUpdater } = require('electron-updater');
 
+const reloadShortCuts = [
+	'CommandOrControl+R',
+	'CommandOrControl+Shift+R',
+	'F5',
```
Comment (yannhodiesne) [inline index.js:19]: I don't think F5 triggers a reload inside the official Discord app

**Review by yannhodiesne — CHANGES_REQUESTED** [review]
Thanks for the fixes!
I can confirm hiding the download button and its separator is working again, without making the whole app crash
The Ctrl/Cmd+R shortcut is a nice touch! Could you bind it only to Ctrl/Cmd+R and Ctrl/Cmd+Shift+R (and not F5) to match the official client behavior?

File: index.js (hunk 1 above)
Comment (BE-CH) [reply index.js]: Fixed in commit https://github.com/yannhodiesne/Discord-M1/pull/17/commits/6cd53a4b4f27ffd687d14444fd29d4f6a32cc2a2

**Comment by BE-CH** [comment]
> Thanks for the fixes! I can confirm hiding the download button and its separator is working again, without making the whole app crash The Ctrl/Cmd+R shortcut is a nice touch! Could you bind it only to Ctrl/Cmd+R and Ctrl/Cmd+Shift+R (and not F5) to match the official client behavior?

Fixed in commit https://github.com/yannhodiesne/Discord-M1/pull/17/commits/6cd53a4b4f27ffd687d14444fd29d4f6a32cc2a2

**Review by yannhodiesne — APPROVED** [review]

**Pull request merged**